audit-service/src/component-tests/java/uk/gov/gchq/palisade/component/audit/config/ApplicationConfigurationTest.java

```diff
@@ -42,10 +42,9 @@
 @SpringBootTest
 @RunWith(SpringRunner.class)
 @ContextConfiguration(classes = ApplicationConfiguration.class)
-@ActiveProfiles(profiles = "")
 public class ApplicationConfigurationTest {
```

Comment:
Noticed that we have some tests updated to JUnit 4 and others updated to JUnit 5.  Expect that we will address this in the stream migration of each service.  Also noticed that we are using a deprecated method isIn in this class

Reply:
removed and reformatted

Reply:
I had a couple of JUnit4/JUnit5 problems, and we have some differences now the tests are in the same maven project. Some tests may change, some may not